pydantic_ai_slim/pydantic_ai/agent.py

```diff
@@ -387,7 +388,7 @@ async def run(
         self,
         user_prompt: str | Sequence[_messages.UserContent] | None = None,
         *,
-        output_type: type[RunOutputDataT] | ToolOutput[RunOutputDataT] | None = None,
+        output_type: type[RunOutputDataT] | ToolOutput[RunOutputDataT] | StructuredOutput[RunOutputDataT] | None = None,
```

Comment:
maybe we should make an alias for this:

```
type Output[T] = type[T] | ToolOutput[T] | StructuredOutput[T]
```
you'd need to use `typing_extensions.TypeAliasType` for backwards compatibility with older python versions but the goal would be to have `Output` be a type alias with the same semantics implied by the above.

(Could use a different name than `Output` but you get the point.)

I'm not sure if it's better, but I think if we tack any more items onto this union, it might be more readable to have a single alias.